### DISCUSSION TITLE
麥當勞內用環保杯

### DISCUSSION THREAD

**Discussion by Jinyuu17** [1101-GS4719-C/Python-Community-GS4719-C#2163] [General]
最近發現越來越多麥當勞在推行內用環保杯，麥當勞的內用環保杯是用塑膠製造的，塑膠的壽命相當的短，而且非常不耐用，大力咬一下或被什麼東西刮到，就會流下刻痕，這些都是細菌生長的絕佳環境。
我覺得真的要環保的話，還是帶自己的杯子去吧。除了更加的衛生，還由額外的五元回饋。若我沒帶杯子，會請店員用外帶杯裝，避免上面提到的麻煩。你們會像我一樣嗎？

**Comment by 112121py** [comment]
基本上我也傾向自己帶環保杯，麥當勞環保杯還有一個問題，他們準備的數量不足以應付高峰時期，導致店員沒有時間洗結果還是只能用外帶杯裝，或是根本洗不乾淨，這部分在網路上都看到蠻多人有反應，他們的環保杯裝飲料上面浮了一層油，在衛生上其實也蠻有疑慮的，只能說想法是好的，但後續的部分沒有做到太讓大部分人滿意(?

**Comment by zhilin93517** [comment]
我比較喜歡帶自己的杯子去，畢竟不是每個人的衛生習慣都相同，對於一些有潔癖的人，他們可能很難接受與別人共用同一個杯子，即使那個杯子有被清洗過。但最吸引我帶自己杯子的理由還是因為有折扣

**Comment by Mminniehuang** [comment]
記得之前在某堂課上老師有和我們討論過關於內用杯的問題（但我有點忘記是不是麥當勞），我記得當時老師說雖然大家一直抨擊衛生問題，不過其實在清洗上一直有一套很完整的流程，包括檢視明顯髒污、放進機器清洗、高溫殺菌等等，可能甚至比自己洗的還乾淨！
不過這也是理論上的狀況，已使用杯子未及時清洗、材質、刮痕污垢確實都會影響，因此我也比較傾向還是自己帶環保杯，既響應他們最初的理念，也真正的安心。

**Comment by wcyeee** [comment]
我也會傾向自己帶！有時候因為對店家的印象，在店員詢問要不要使用環保餐具的時候反而會因為衛生問題而選擇還是要用一次性餐具（然後就會覺得不太好意思），所以最後還是乾脆都自己帶。

**Comment by lai1008** [comment]
我比較喜歡帶自己的杯子去因為非常環保，但最吸引我帶自己杯子的理由還是因為有折扣

**Comment by Boying12** [comment]
對我來說，外帶杯的選擇更為方便，也不需要額外攜帶杯子。環保固然重要，但在日常生活中，便利性和實用性同樣值得考慮。這樣的方式也讓我能夠輕鬆地享受美食，而不必擔心杯子的問題。

**Comment by NiCL100** [comment]
這感覺就是經濟跟環保到衝突，其實如果麥當勞願意的話，大可以用其他材質做內用杯，不過，商人唯一感興趣的就是利益，塑膠內用杯也不知道可以用多久，真的有環保嗎？可能我們還是自己攜帶杯子才最環保吧，但我會這麼麻煩當然主要是因為折扣。

**Comment by qiu0109** [comment]
麥當勞的環保杯感覺都洗不太乾淨，我覺得維持之前的外帶杯會比較好，而且麥當勞的飲料分量不小，每次吃完飲料都還會剩半杯，用外帶杯比較方便吃完之後把飲料帶走，當然最好的選擇還是自己帶環保杯。

**Comment by lichilun** [comment]
我能同理你的擔憂，因為我也有這樣的想法，塑膠杯使用後不耐用，容易留下刻痕，滋生細菌。以環保的角度來看，帶自己的杯子不僅可以減少塑膠的使用，還能更加衛生，像你提到的，麥當勞的五元回饋，聽起來是一個不錯的激勵措施。
